spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Graph.scala

```diff
@@ -72,8 +71,8 @@ import org.tensorflow.framework.GraphDef
 @SerialVersionUID(- 2896121321564992779L)
 abstract class Graph[T: ClassTag](
   val inputs : Seq[ModuleNode[T]],
-  protected val outputs : Seq[ModuleNode[T]],
-  private val variables: Option[(Array[Tensor[T]], Array[Tensor[T]])] = None
+  private[bigdl] val outputs : Seq[ModuleNode[T]],
+  private[bigdl] val variables: Option[(Array[Tensor[T]], Array[Tensor[T]])] = None
```

Comment:
why change the scope of the variables and outputs?

Reply:
`variables` or `variables` cannot be access by  `GraphSerializable` from the original scope as it's extracted from object `Graph`. Not sure if there's any other good solution for this, or I guess we can change the scope if no specific concern.